pkg/channel/test/message.go

```diff
@@ -1,19 +1,25 @@
 package test
 
 import (
-	"github.com/confluentinc/confluent-kafka-go/kafka"
+	"github.com/Shopify/sarama"
+	"github.com/cloudevents/sdk-go/v2/binding"
 	"github.com/stretchr/testify/assert"
 	"testing"
 )
 
+// Utility Function For Creating A CloudEvents sdk-go BindingMessage
+func CreateBindingMessage(cloudEventVersion string) binding.Message {
+	return binding.ToMessage(CreateCloudEvent(cloudEventVersion))
```

Comment:
There are some test utils in `github.com/cloudevents/sdk-go/v2/test` that you might find interesting

Reply:
It would be good to re-use these but at the moment they are based on hard-coded values instead of exposed constants which would make my tests more brittle.  It's great to know they're there and I will keep them in mind and possibly contribute some hardening so that we could consume in the future.